src/components/collapsible/Collapsible.js

```diff
@@ -37,10 +36,19 @@ type Props = {
 };
 
 type State = {
+    /** This is the height of the animated div */
+    height: string,
+    /** This value represents if the collapsible is open or not */
     isOpen: boolean,
 };
 
-class Collapsible extends React.Component<Props, State> {
+type Ref = {
+    current: null | HTMLElement,
```

Comment:
I believe this can just be ?HTMLDivElement and you can inline it within the class if you want since its used at one place only.